docs/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "gridplus-sdk",
-  "version": "0.0.0",
+  "name": "gridplus-sdk-docs",
+  "version": "0.1.0",
```

Comment:
🤔

Reply:
yeah--the docs package had the same name as the lib 😄

Reply:
And I figured we might as well tick up the version as we make changes. Doesn't really matter too much.